source/includes/common-replication.rst

```diff
@@ -141,6 +141,12 @@ Each MinIO deployment ("peer site") synchronizes the following changes across th
 
 Site replication enables :ref:`bucket versioning <minio-bucket-versioning>` for all new and existing buckets on all replicated sites.
 
+.. versionadded:: mc RELEASE.2023-12-02T02-03-28Z
+
+You can choose to replicate ILM expiration rules across peer sites.
+Enable this option either at the time of configuring the site replication or by modifying the existing site replication configuration.
```

Comment:
```suggestion
You can enable this option using the :mc-cmd:`mc replicate add --replicate-ilm-expiry` modifier at the time of configuring the site replication. 
You can also enable or disable the behavior on an existing sit e replication configuration using :mc-cmd:`mc replicate edit`.
```

Reply:
Tweaked it and pointed to the `mc admin replicate` command instead, but otherwise 👍